2019-Container/Packages/packages-lock.json

```diff
@@ -8,41 +8,67 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.ext.nunit": {
-      "version": "1.0.0",
+      "version": "1.0.6",
       "depth": 1,
       "source": "registry",
       "dependencies": {},
       "url": "https://packages.unity.com"
     },
     "com.unity.ide.rider": {
-      "version": "1.1.4",
+      "version": "1.2.1",
       "depth": 0,
       "source": "registry",
       "dependencies": {
         "com.unity.test-framework": "1.1.1"
       },
       "url": "https://packages.unity.com"
     },
+    "com.unity.ide.visualstudio": {
```

Comment:
Do we want this?

Reply:
this is something visual studio add to the testapp